datahub-upgrade/src/test/java/com/linkedin/datahub/upgrade/system/debezium/WaitForDebeziumReadyStepTest.java

```diff
@@ -0,0 +1,112 @@
+package com.linkedin.datahub.upgrade.system.debezium;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+import com.linkedin.datahub.upgrade.system.cdc.debezium.WaitForDebeziumReadyStep;
+import com.linkedin.metadata.config.DebeziumConfiguration;
+import com.linkedin.metadata.config.kafka.KafkaConfiguration;
+import io.datahubproject.metadata.context.OperationContext;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class WaitForDebeziumReadyStepTest {
```

Comment:
Doesn't appear to be a duplicate, but this test has the exact same name? Can they be combined/made more specific to avoid confusion?